src/utils/makeReactNativeConfig.js

```diff
@@ -67,6 +90,7 @@ const getDefaultConfig = (options: EnvOptions): WebpackConfig => {
     providesModuleNodeModules,
     hasteOptions,
     disableHotReloading,
+    includedNodeModules,
```

Comment:
Sorry, I'm still a Webpack noob. It seems that we're preventing these modules from being transpiled by babel-loader. My concern is that it's not obvious what this option does without having to dig into the code. Can we rename it to something more specific, e.g. `babelLoaderExcludedModules`?

@thymikee, what do you think?

Reply:
I think I would be more keen on exposing the helper and document how to use it instead of introducing a new option

Reply:
Just for my understanding as an outsider: would you mind elaborating on your thinking about the trade-offs here?